Tests/MongoSwiftSyncTests/SpecTestRunner/SpecTest.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
```

Comment:
I think this file should also be split up into several files, but for the sake of this diff I opted to not do it as part of this PR.